### PR TITLE
feat: add `create` and `new` alias for `init`

### LINKED ISSUE #2599: [Feature]: Rename init to create
**Is your feature request related to a problem? Please describe.**

Init isn't a very good name for creating/scaffolding new configurations.

**Describe the solution you'd like**

Rename `webpack-cli` init to `webpack-cli create`

**Describe alternatives you've considered**

Nothing

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
Yes WIP
**Summary**
Fixes #2599 

create new aliases - `create`, `new`

Latest usage - `webpack init|create|new|c|n [generation-path] [options]`

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by alexander-akait — COMMENTED** [review]
Can we update `OPTIONS.md`?

**Comment by snitin315** [comment]
Yes just fixing one issue, now init is printing three times in help output.

Screenshot 2021-04-12 at 4 32 54 PM

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ class WebpackCLI {
 
     async makeCommand(commandOptions, options, action) {
         const alreadyLoaded = this.program.commands.find(
-            (command) => command.name() === commandOptions.name || command.aliases().includes(commandOptions.alias),
+            (command) =>
+                command.name() === commandOptions.name ||
+                command.aliases().includes(commandOptions.alias) ||
+                JSON.stringify(command.aliases()) === JSON.stringify(commandOptions.alias),
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/webpack-cli.js:30]: Bad for performance

Comment (snitin315) [reply packages/webpack-cli/lib/webpack-cli.js:30]: We have command.name() = "init",
commandOptions.name = "init [generation-path]"

How about the following? 

```suggestion
                command.name() === commandOptions.name.split(' ')[0],
```

**Comment by alexander-akait** [comment]
I will finish PR, thanks for help

**Comment by alexander-akait** [comment]
Found other bug, `webpack help new` is not working

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**